src/main/java/racingcar/Racing.java

```diff
@@ -4,24 +4,44 @@
 import java.util.List;
 
 public class Racing {
-    private List<Car> cars = new ArrayList<>();
+    private List<Car> readyCars = new ArrayList<>();
 
     public List<Car> getCars(){
-        return this.cars;
+        return this.readyCars;
     }
 
-    public void readyCar(int CarCount){
-        for(int i = 0; i < CarCount; i++){
-            cars.add(new Car());
+    public void readyCar(String[] cars){
+        for(int i = 0; i < cars.length; i++){
+            readyCars.add(new Car(cars[i], 0));
         }
     }
 
     public List<Car> moveCars(){
-        for(Car car : cars){
+        for(Car car : readyCars){
             Forward forward = new Forward();
             int forwardValue = forward.moveForwardValue();
             car.moveForward(forwardValue);
         }
-        return cars;
+        return readyCars;
+    }
+
+    public Winners getWinners(){
+        Winners winners = new Winners();
+        for(Car car : readyCars){
+            addWinners(car, winners);
+        }
+        return winners;
+    }
+    public void addWinners(Car car, Winners winners){
+        if(car.getPosition() == getMaxPosition())
+            winners.addWinner(car);
```

Comment:
자바 코드 컨벤션을 지켜주도록 해주세요🙏
```suggestion
        if (car.getPosition() == getMaxPosition()) { 
            winners.addWinner(car);   
        }
```